### PR TITLE
Update for Storage commands

### PR DESCRIPTION
**Pull request by hasonmsft**
The content to be added to Changelog is as follows:
* Upgraded azure-common package to 0.9.17 and request package to 2.74.0 to address the security issue and proxy tunneling issue. 
* Tuned the description of the `--snapshot` option for blob commands.
* Tuned the description of the storage account credential related options for storage commands. 
* Added the command `azure storage blob update` to update the properties of an existing blob.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     "progress": "^1.1.8",
     "prompt": "0.2.14",
     "readable-stream": "~1.0.0",
-    "request": "2.69.0",
+    "request": "2.72.0",
```
Comment (haocs) [inline package.json]: request v2.74.0 is the version that fixed though-cookie issue. Please update it to 2.74.0

Comment (hasonmsft) [inline package.json]: Thanks for the confirmation. I'll update the PR.

File: lib/commands/storage/storage.blob._js [hunk 2]
```diff
@@ -1157,6 +1170,63 @@ exports.init = function(cli) {
...
+      properties.contentType = specifiedProperties.contenttype;
+      properties.contentEncoding = specifiedProperties.contentencoding;
+      properties.contentLanguage = specifiedProperties.contentlanguage;
+      properties.cacheControl = specifiedProperties.cachecontrol;
+      properties.contentDisposition = specifiedProperties.contentdisposition;
+      properties.contentMD5 = specifiedProperties.contentmd5;
+
+      // Merge with existing properties on the server so that user can choose to update the properties partially instead of overwritting all the properties everytime.
+      if (existingBlobProperties && existingBlobProperties.contentSettings) {
+        properties.contentType = properties.contentType !== undefined ? properties.contentType : existingBlobProperties.contentSettings.contentType;
```
Comment (haocs) [inline lib/commands/storage/storage.blob._js]: Is it acceptable if `properties.contentType` is `null`?

**Comment by haocs** [comment]
@hasonmsft , Left one more comment. others look good to me!

File: lib/commands/storage/storage.blob._js (hunk 2 above)
Comment (hasonmsft) [inline lib/commands/storage/storage.blob._js]: No. How can user supply `null` via the CLI prompt framework?

**Comment by haocs** [comment]
LGTM

**Comment by yaxia** [comment]
Note: This addressed #3034.

**Pull request merged**